cypress/integration/podcasts.spec.js

```diff
@@ -2,7 +2,8 @@
 
 const homeurl = "http://localhost:3000"
 
-const switchPodCast = (podcast) => {
+const switchPodCast = (podcast, add) => {
+  if (add == true) cy.get("#select-podcast").eq(0).click()
```

Comment:
not sure what add really means here, its not adding anything, maybe a better var name?
also if you're going to do an if it should be `===`